### ISSUE TITLE
Loading Assets, At All

### ISSUE THREAD

**Issue by cScarlson** [balderdashy/sails-docs#487]
Hi,

At this point I'm certain that my code *should* work, but its not. This is really basic -- all I'm trying to do is load a script *using a script tag*. I've initialized the project with `sails new api2`, added a `assets/alert.js` to `assets`, and added `script` tags to load `js/dependencies/sails.io.js` & `/js/alert.js` to my `layout.ejs` file. However, when I run `sails lift` and go out to `http://localhost:1337` -- I get a `404` on both scripts.

	<script src="js/dependencies/sails.io.js" type="text/javascript"></script>
	<script src="js/alert.js" type="text/javascript"></script>

The doc at `http://sailsjs.org/#!/documentation/anatomy/myApp/assets` says:
<blockquote>
"
This is your assets folder. It houses all of the static files that your app will need to host. Feel free to create your own files and folders in here. Upon lifting, a file called myApp/assets/newFolder/data.txt could be accessed at http://localhost:1337/newFolder/data.txt.
"
</blockquote>
But this is simply *not true*.

I'm certain that I'm not doing anything wrong -- or at least nothing complex -- and **I cannot get the most basic of an application together**.

I'm I doing something absurd, or is this a bug?

If a config is necessary just to load script-tags, my confidence in Sails is blown -- but I'm certain I'm missing something.

**Comment by jub3i** [comment]
you are making silly mistake :)

since you put `alert.js` file here `assets/alert.js` you can load it with script tag like this: `<script src="/alert.js" type="text/javascript"></script>`

Previous method was manual way. If you put `alert.js` in `assets/js/alert.js` then appropriate `<script>` tag will be automatically injected into your layout file on next `sails lift`, which you can verify at bottom of `views/layout.ejs` (after you do a `sails lift` again!).

read more at http://sailsjs.org/#!/documentation/concepts/Assets/DefaultTasks.html?q=sailslinker

**Comment by cScarlson** [comment]
Oops! I did make a silly mistake -- a type-o. All my `src` paths are correct -- someone on Stackoverflow pointed out that there was a recent bug (~ v0.11.n) which was, indeed, the issue:

<blockquote>
"@Cody if you just downloaded Sails recently, there was a bug for a few days in the sails new which caused the normal asset pipeline not to work. Open up your .sailsrc file and if you see "hooks": {"grunt": false}, remove it. The next time you lift your Sails app it should work with your script src pointing to /js/dependencies/sails.io.js. –  sgress454 22 hours ago "
</blockquote>

I followed these directions and now I'm free as a bird -- all source-paths work as expected.
